.github/workflows/push.yaml

```diff
@@ -45,3 +45,37 @@ jobs:
           tag: ${{ github.sha }}
           expiration: +1 week
           token: ${{ secrets.QUAY_OAUTH_TOKEN }}
+
+  is-backstage-updated:
+    runs-on: ubuntu-latest
+    outputs:
+      backstage-bump: ${{ steps.changes.outputs.backstage-bump }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - uses: dorny/paths-filter@v2
+        id: changes
+        with:
+          filters: |
+            backstage-bump:
+              - 'backstage.json'
```

Comment:
Can you make this a separate workflow with the GH native path filter?

Reply:
Will do.

Reply:
Looks much cleaner now, changed in the latest cmmit.